src/popup/components/Modals/ConsensusInfo.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <Default
+    v-bind="{ ...$attrs, resolve }"
+    icon="info"
+    :title="$t('modals.consensus-info.title')"
+    :close="resolve"
+  >
+    <template #msg>
+      <div class="content">
+        <div class="subtitle">
+          {{ $t("modals.consensus-info.subtitle") }}
+        </div>
+        <ul class="list">
+          <li>
+            <TemplateRenderer
+              :str="$t('modals.consensus-info.first-point')"
+            />
+          </li>
+          <li>
+            <TemplateRenderer
+              :str="$t('modals.consensus-info.second-point')"
+            />
+          </li>
+          <li>
+            <TemplateRenderer
+              :str="$t('modals.consensus-info.third-point')"
+            />
+          </li>
+        </ul>
+
+        <TemplateRenderer
+          :str="$t('modals.consensus-info.description')"
+        />
+      </div>
+    </template>
+    <template #footer>
+      <BtnMain
+        class="footer"
+        extend
+        @click="resolve"
+      >
+        {{ $t("ok") }}
+      </BtnMain>
+    </template>
+  </Default>
+</template>
+
+<script>
+import {} from 'vue-i18n';
+import Default from './Default.vue';
+import BtnMain from '../buttons/BtnMain.vue';
+import { BLOG_CLAIM_TIP_URL } from '../../utils/constants';
+import TemplateRenderer from '../TemplateRenderer.vue';
+
+export default {
+  components: {
+    TemplateRenderer,
+    Default,
+    BtnMain,
+  },
+  props: {
+    resolve: { type: Function, required: true },
+    close: { type: Function, default: null },
+  },
+  data: () => ({
+    BLOG_CLAIM_TIP_URL,
+    UNFINISHED_FEATURES: process.env.UNFINISHED_FEATURES,
+  }),
+};
+</script>
+
+<style lang="scss" scoped>
+@use "../../../styles/share-info";
```

Comment:
This file is written in a really bad way - mostly because it applies styling onto html elements instead of using CSS classes. I strongly discourage you from importing it.